chunjun-connectors/chunjun-connector-jdbc-base/src/main/java/com/dtstack/chunjun/connector/jdbc/sink/JdbcDynamicTableSink.java

```diff
@@ -70,7 +69,6 @@ public ChangelogMode getChangelogMode(ChangelogMode requestedMode) {
                 .addContainedKind(RowKind.INSERT)
                 .addContainedKind(RowKind.DELETE)
                 .addContainedKind(RowKind.UPDATE_AFTER)
-                .addContainedKind(RowKind.UPDATE_BEFORE)
```

Comment:
Why delete the 'UPDATE_BEFORE'?

Reply:
Use upsert statements to handle -U and +U